Sources/AppCoinsSDK/Models/Services/Billing/AppCoinBillingClient.swift

```diff
@@ -553,4 +553,32 @@ internal class AppCoinBillingClient : AppCoinBillingService {
         }
     }
     
+    internal func getSupportedCurrencies(result: @escaping (Result<CurrencyListRaw, BillingError>) -> Void) {
```

Comment:
We should deal with pagination. The request returns `next` and `previous` pointers, so we should query the rest of the currencies if next is not `null`.

Reply:
Done